bin/start_be.sh

```diff
@@ -16,11 +16,31 @@
 # specific language governing permissions and limitations
 # under the License.
 
-curdir=$(dirname "$0")
-curdir=$(
-    cd "$curdir"
-    pwd
-)
+# resolve links - $0 may be a softlink
+PRG="$0"
+
+while [ -h "$PRG" ] ; do
```

Comment:
Consider command `readlink` to resolve the soft link problem?

Reply:
yes. this method has been used in my project for a long time, you can see: https://github.com/apache/tomcat/blob/main/bin/startup.sh